### DISCUSSION TITLE
google sheet for translating .dst file stitches

### DISCUSSION THREAD

**Discussion by rlevine** [inkstitch/inkstitch#2146] [General]
I've been spending time staring at hex dumps of .dst files, so I created a spreadsheet to help me see what's going on in them.

Google link is [here](https://docs.google.com/spreadsheets/d/1JetcNxZG_bJRxaLnaqiZPbq1-U6SKY-tYLr5eJKRoyw/edit?usp=sharing)

To use it, dump the binary stitch data, skipping the header, from a .dst file by running xxd on the file in a shell:

`xxd -s512 -c3 -b <filename>.dst`

Copy the results and paste them into into cell A9 ofthe sheet, and then convert the date to columns by choosing menu: Data -> Split text to columns.

The sheet will deal with 1000 stitches; it's easiest to write smaller tests, otherwise you'll need to fish around in the data to find the stitches you want to look at.

Rick

***

<img width="1503" alt="Screen Shot 2023-03-15 at 12 44 55 PM" src="https://user-images.githubusercontent.com/2294569/225425433-aea6021a-b7cc-4af8-8941-c4a1fdceb398.png">

**Comment by lexelby** [comment]
Wow!
